yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -932,7 +932,7 @@ func getUnsupportedFeaturesFromSchemaAnalysisReport(featureName string, issueDes
 	var minVersionsFixedInSet bool
 
 	for _, analyzeIssue := range schemaAnalysisReport.Issues {
-		if !slices.Contains([]string{UNSUPPORTED_FEATURES_CATEGORY, MIGRATION_CAVEATS_CATEGORY}, analyzeIssue.IssueType) {
+		if !slices.Contains([]string{UNSUPPORTED_FEATURES_CATEGORY, MIGRATION_CAVEATS_CATEGORY, PERFORMANCE_OPTIMIZATIONS_CATEGORY}, analyzeIssue.IssueType) {
```

Comment:
just curious, why do we have this check? we already filter by `analyzeIssue.Type`, right? why do we need this additional check to exclude a few issues?

Reply:
got it, pls add a comment there. 
Also, checking and filtering out the PLPGSQL issues might be more readable.